BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsEventParams.kt

```diff
@@ -4,10 +4,13 @@ import androidx.annotation.RestrictTo
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 data class AnalyticsEventParams constructor(
-    var payPalContextId: String?,
-    var linkType: String?,
-    var isVaultRequest: Boolean,
+    var payPalContextId: String? = null,
+    var linkType: String? = null,
+    var isVaultRequest: Boolean = false,
+    var startTime: Long = -1,
+    var endTime: Long = -1,
```

Comment:
Can we make this nullable and default this to `null` instead of `-1`? Similar to the `linkType`?

Reply:
Yes we can, addressed here, let me know what do you think: [1a32ee9](https://github.com/braintree/braintree_android/pull/1032/commits/1a32ee9f0998d5de4fd6e5c4ae5a539ceb554c49)